Source/WebCore/page/Page.h

```diff
@@ -243,6 +243,7 @@ enum class RenderingUpdateStep : uint32_t {
     AccessibilityRegionUpdate       = 1 << 20,
 #endif
     RestoreScrollPositionAndViewState = 1 << 21,
+    UpdateContentRelevancy          = 1 << 22,
```

Comment:
Please move this above IntersectionObservations.

Reply:
Sure, moved.